src/index.html

```diff
@@ -4,9 +4,417 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a class="logo_link" href="#home">
+          <img
+            src="/images/logo.png"
+            alt="MOYO logo"
+            class="logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>
            
            <li class="nav__item">
              <a href="#samsung" class="nav__link">Samsung</a>
            </li>
```
add indents between blocks of code